app/connectors/BaseConnector.scala

```diff
@@ -43,4 +56,28 @@ class BaseConnector extends HttpErrorFunctions {
       .withExtraHeaders(extraHeaders: _*)
     newHeaderCarrier
   }
+
+  def post(messageType: String, message: String, itemId: ItemId)(implicit hc: HeaderCarrier, ec: ExecutionContext): Future[HttpResponse] = {
+    val url = itemId match {
+      case ArrivalId(index) =>
+        appConfig.traderAtDestinationUrl + arrivalRoute format (index, Constants.MessageCorrelationId)
+      case DepartureId(index) =>
+        appConfig.traderAtDeparturesUrl + departureRoute format (index, Constants.MessageCorrelationId)
```

Comment:
The route constants could be moved to the Constants.scala file.

Reply:
> But previously those urls were constructed like this:
> `val eisPath = arrivalRoute format (arrivalId.index, Constants.MessageCorrelationId) val url = appConfig.traderAtDestinationUrl + eisPath`
> so does it mean that now it will be fine just to leave
> `arrivalRoute format (index, MessageCorrelationId)` ?

The lines in the PR are correct as they are now. So no need to change this 👍 .